build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -3,8 +3,16 @@
   <Import Project="compile/Microsoft.DotNet.Cli.LzmaArchive.targets" />
   <Import Project="crossgen/Microsoft.DotNet.Cli.Crossgen.targets" />
 
+  <PropertyGroup>
+    <BinaryToCorehostifyRelDir>runtimes/any/native</BinaryToCorehostifyRelDir>
+    <CoreSDKDir>$(RepoRoot)/resources/MSBuildImports</CoreSDKDir>
```

Comment:
Same for $(CoreSDKContent) below.

---
In reply to: [74152883](https://github.com/dotnet/cli/pull/4024#discussion_r74152883) [](ancestors = 74152883)